demos/_unit-test/scope-builder.php

```diff
@@ -72,23 +77,30 @@
                 "query": {
                   "rule": "project_name",
                   "operator": "matches regular expression",
-                  "value": "[a-zA-Z]"
+                  "value": "[a-zA-Z]",
+                  "option": null
                 }
               },
               {
                 "type": "query-builder-rule",
                 "query": {
                   "rule": "client_country_iso",
                   "operator": "equals",
-                  "value": "Brazil"
+                  "value": "30",
+                  "option": {
+                    "key": "30",
```

Comment:
What's the difference between key and value? And i think it would be better to use name `label` instead of `text` if that's what it means. Why this `option` block is needed at all?

Reply:
Option block is use to format atkLookup component correctly when a value is set. In other word, the atk-lookup component required the option key, text and value to be properly set and display. 

As of now, only atk-lookup required this, but in future, it could be a range slider component that would require min or max value for example. These would be pass via the option property.

Reply:
Can't these be pulled from respective data model not saved in json? For example, what if we rename Brazil to Argentina in some other form? Will this json will still be valid then?

Reply:
It will still be working because the option is generated on load. Also, option is irrelevant when doing query to scope.

Reply:
Option was needed for vue builder. It has no meaning (to store it) in our case, but removing of it from vue requires a lot of changes in vue class, so we can just ignore it on our side. It's reloaded on widget load with proper values anyway.